paddle/fluid/API.spec

```diff
@@ -5,6 +5,7 @@ paddle.fluid.Program.current_block ArgSpec(args=['self'], varargs=None, keywords
 paddle.fluid.Program.global_block ArgSpec(args=['self'], varargs=None, keywords=None, defaults=None)
 paddle.fluid.Program.list_vars ArgSpec(args=['self'], varargs=None, keywords=None, defaults=None)
 paddle.fluid.Program.parse_from_string ArgSpec(args=['binary_str'], varargs=None, keywords=None, defaults=None)
+paddle.fluid.Program.prune_read_op_and_convert_for_test ArgSpec(args=['self'], varargs=None, keywords=None, defaults=None)
```

Comment:
It can be a start of a suite of prune_xxx_op...
prune_send_op
prune_lr_op
prune_op_I_dont_like...

I will be nightmare

Reply:
Remove it. Done.